driver/ppm_fillers.c

```diff
@@ -2694,9 +2694,9 @@ static int timespec_parse(struct event_filler_arguments *args, unsigned long val
 {
 	uint64_t longtime;
 	char *targetbuf = args->str_storage;
-	struct timespec *tts = (struct timespec *)targetbuf;
+	struct old_timespec32 *tts = (struct old_timespec32 *)targetbuf;
```

Comment:
I am not sure about his one.

Reply:
I think this should be `timespec64` but only on 64-bit. On 32-bit this is `timespec32`.

Reply:
Is `__kernel_timespec` the right choice here?

Reply:
In that case sysdig needs to use: `get_timespec64()`

Reply:
Just read the docs (ioct.rst, "Timestamps") and ``get_timespec64()`` seems like the right way.

Reply:
`timespect` sounds like `__kernel_old_timespec` to me.